cypress/support/fragments/quickMarcEditor.js

```diff
@@ -1531,4 +1532,14 @@ export default {
   clickUnlinkButton: () => {
     cy.do(buttonLink.click());
   },
+
+  checkDefaultContent() {
+    this.checkContent('00000n\\\\\\a2200000uu\\4500', 0);
+    this.checkFieldsExist(expectedFields);
+    this.checkEmptyContent('001');
+    this.checkEmptyContent('005');
+    this.checkEmptyContent('999');
+    this.checkEmptyContent('008');
+    this.verifyTagField(4, '245', '\\', '\\', '$a ', '');
```

Comment:
Could you please also add a check with `checkContent()` method for 245 field to make sure that "$a " is the only thing that is shown there (`verifyTagField()` uses `including` matcher, so it will pass even if there is some value after $a or even other subfields)?